requirements.txt

```diff
@@ -1,22 +1,18 @@
 cleanlab
```

Comment:
Add comment next to this: 

#  Some examples may require you to instead install latest developer version of cleanlab from github:  `pip install git+https://github.com/cleanlab/cleanlab.git`